File: specification/specs/x_pack/machine_learning/job/results/BucketInfluencer.ts
class BucketInfluencer {
	bucket_span: long;
	influencer_field_name: string;
	influencer_field_value: string;
	influencer_score: double;
	initial_influencer_score: double;
	is_interim: boolean;
	job_id: string;
	probability: double;
	result_type: string;
	@prop_serializer("DateTimeOffsetEpochMillisecondsFormatter")
	timestamp: Date;
}

File: output/csharp/XPack/MachineLearning/Job/Results/BucketInfluencer.cs
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class BucketInfluencer  {
		
		[DataMember(Name="bucket_span")]
		public long BucketSpan { get; set; }


		[DataMember(Name="influencer_field_name")]
		public string InfluencerFieldName { get; set; }


		[DataMember(Name="influencer_field_value")]
		public string InfluencerFieldValue { get; set; }


		[DataMember(Name="influencer_score")]
		public double InfluencerScore { get; set; }


		[DataMember(Name="initial_influencer_score")]
		public double InitialInfluencerScore { get; set; }


		[DataMember(Name="is_interim")]
		public bool IsInterim { get; set; }


		[DataMember(Name="job_id")]
		public string JobId { get; set; }


		[DataMember(Name="probability")]
		public double Probability { get; set; }


		[DataMember(Name="result_type")]
		public string ResultType { get; set; }


		[DataMember(Name="timestamp")]
		public DateTimeOffset Timestamp { get; set; }

	}
}
